pkg/tsdb/mssql/mssql.go

```diff
@@ -157,14 +154,17 @@ func generateConnectionString(dsInfo sqleng.DataSourceInfo) (string, error) {
 	if addr.Port != "0" {
 		connStr += fmt.Sprintf("port=%s;", addr.Port)
 	}
+	if encrypt == "true" {
+		connStr += fmt.Sprintf("encrypt=%s;TrustServerCertificate=%t;", encrypt, tlsSkipVerify)
+		if hostNameInCertificate != "" {
+			connStr += fmt.Sprintf("hostNameInCertificate=%s;", hostNameInCertificate)
+		}
 
-	if dsInfo.JsonData.Encrypt == "" {
-		dsInfo.JsonData.Encrypt = "false"
-	}
-
-	if dsInfo.JsonData.Encrypt != "false" {
-		connStr += fmt.Sprintf("encrypt=%s;", dsInfo.JsonData.Encrypt)
+		if certificate != "" {
+			connStr += fmt.Sprintf("certificate=%s;", certificate)
+		}
 	}
+	logger.Info("connection", "value", connStr)
```

Comment:
Please remove. Can be sensitive information here. We already have a log of the connectionstring if running in dev mode, see https://github.com/grafana/grafana/blob/1427043360ac19543c31f7e9a180a69f9fb4f949/pkg/tsdb/mssql/mssql.go#L82-L84

Reply:
done